src/System.Memory/tests/Performance/Perf.Utf8Parser.cs

```diff
@@ -99,32 +526,99 @@ private static void ParserInt16(string text)
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        private static void ByteSpanToInt16_VariableLength_Baseline()
+        {
+            int textLength = s_Int16TextArray.Length;
+            byte[][] utf8ByteArray = (byte[][])Array.CreateInstance(typeof(byte[]), textLength);
+            for (var i = 0; i < textLength; i++)
+            {
+                utf8ByteArray[i] = Encoding.UTF8.GetBytes(s_Int16TextArray[i]);
+            }
+
+            foreach (BenchmarkIteration iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
+                        short.TryParse(s_Int16TextArray[i % textLength], out short value);
+                        TestHelpers.DoNotIgnore(value, 0);
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData("True")]
+        [InlineData("False")]
+        private static void StringToBool_Baseline(string text)
+        {
+            foreach (BenchmarkIteration iteration in Benchmark.Iterations)
+            {
+                using (iteration.StartMeasurement())
+                {
+                    for (int i = 0; i < Benchmark.InnerIterationCount; i++)
+                    {
+                        bool.TryParse(text, out bool value);
+                    }
+                }
+            }
+        }
+
+        [Benchmark(InnerIterationCount = InnerCount)]
+        [InlineData("True")]
+        [InlineData("False")]
+        private static void BytesSpanToBool(string text)
         {
             byte[] utf8ByteArray = Encoding.UTF8.GetBytes(text);
             ReadOnlySpan<byte> utf8ByteSpan = utf8ByteArray;
-
             foreach (BenchmarkIteration iteration in Benchmark.Iterations)
             {
                 using (iteration.StartMeasurement())
                 {
                     for (int i = 0; i < Benchmark.InnerIterationCount; i++)
                     {
-                        Utf8Parser.TryParse(utf8ByteSpan, out ushort value, out int bytesConsumed);
+                        Utf8Parser.TryParse(utf8ByteSpan, out bool value, out int bytesConsumed);
                         TestHelpers.DoNotIgnore(value, bytesConsumed);
                     }
                 }
             }
         }
 
         [Benchmark(InnerIterationCount = InnerCount)]
-        [InlineData("12837467")] // standard parse
-        [InlineData("-2147483648")] // min value
-        [InlineData("2147483647")] // max value
-        private static void ParserInt32(string text)
+        [InlineData("0")]
+        [InlineData("107")] // standard parse
+        [InlineData("127")] // max value
+        [InlineData("-128")] // min value
+        [InlineData("147")]
```

Comment:
We should remove these two test cases (147 and -111) as they don't provide much value.

Reply:
Remove `[InlineData("147")]` and `[InlineData("-111")]`

Reply:
Still waiting to remove these.